File: SeedDocs.py
import pyodbc

conn_str = r'Driver={ODBC Driver 17 for SQL Server};Server=(localdb)\mssqllocaldb;Database=GMK360Db;Trusted_Connection=yes;'
conn = pyodbc.connect(conn_str, autocommit=True)
cursor = conn.cursor()

docs = [
    ('Arsa Tapusu', 'Tapu Müdürlüğü', 'Construction', 1),
    ('İmar Durumu Belgesi', 'Belediye', 'Construction', 1),
    ('Mimari Proje Onayı', 'Belediye İmar Müdürlüğü', 'Construction', 1),
    ('Zemin Etüdü Raporu', 'Özel Zemin/Jeoloji Firması', 'Construction', 1),
    ('Yapı Ruhsatı', 'Belediye', 'Construction', 1)
]

cursor.execute('SELECT COUNT(*) FROM SystemLegalDocumentTemplates')
if cursor.fetchone()[0] == 0:
    for name, issued, target, req in docs:
        cursor.execute('''
            INSERT INTO SystemLegalDocumentTemplates (Name, IssuedBy, TargetModule, IsMandatory, CreatedAt, IsDeleted) 
            VALUES (?, ?, ?, ?, GETUTCDATE(), 0)
        ''', (name, issued, target, req))
    print('Evraklar eklendi.')
else:
    print('Evraklar zaten var.')

File: SeedDocs.cs
using System;
using System.Linq;
using System.Collections.Generic;
using Microsoft.EntityFrameworkCore;
using Microsoft.Extensions.DependencyInjection;
using GMK360.Data.Contexts;
using GMK360.Core.Entities.Admin;

var optionsBuilder = new DbContextOptionsBuilder<ApplicationDbContext>();
optionsBuilder.UseSqlServer(@""Server=(localdb)\mssqllocaldb;Database=GMK360Db;Trusted_Connection=True;MultipleActiveResultSets=true"");

using (var db = new ApplicationDbContext(optionsBuilder.Options))
{
    var existing = db.SystemLegalDocumentTemplates.Any();
    if (!existing)
    {
        var docs = new List<SystemLegalDocumentTemplate>
        {
            new SystemLegalDocumentTemplate { Name = ""Arsa Tapusu"", IssuedBy = ""Tapu Müdürlüğü"", TargetModule = ""Construction"", IsRequired = true },
            new SystemLegalDocumentTemplate { Name = ""İmar Durumu Belgesi"", IssuedBy = ""Belediye"", TargetModule = ""Construction"", IsRequired = true },
            new SystemLegalDocumentTemplate { Name = ""Mimari Proje Onayı"", IssuedBy = ""Belediye İmar Müdürlüğü"", TargetModule = ""Construction"", IsRequired = true },
            new SystemLegalDocumentTemplate { Name = ""Zemin Etüdü Raporu"", IssuedBy = ""Özel Zemin/Jeoloji Firması"", TargetModule = ""Construction"", IsRequired = true },
            new SystemLegalDocumentTemplate { Name = ""Yapı Ruhsatı"", IssuedBy = ""Belediye"", TargetModule = ""Construction"", IsRequired = true }
        };
        db.SystemLegalDocumentTemplates.AddRange(docs);
        db.SaveChanges();
        Console.WriteLine(""Evraklar eklendi."");
    }
    else {
        Console.WriteLine(""Evraklar zaten var."");
    }
}
